src/lightning_app/cli/lightning_cli.py

```diff
@@ -267,6 +269,9 @@ def _run_app(
 
     secrets = _format_input_env_variables(secret)
 
+    port = _find_lit_app_port(constants.APP_SERVER_PORT)
+    constants.APP_SERVER_PORT = port
```

Comment:
If there is really no other way than making it a global variable, then let's move it out of the "constants" file? Clearly, if it changes at runtime, it's no longer a constant.

Perhaps `constants.py` could contain `APP_SERVER_DEFAULT_PORT=7501` and then, I don't know, maybe `lightning_app.py` could hold the global var for the port that's being used.

Reply:
Thanks for the review @awaelchli I will update per your suggestion